.travis.yml

```diff
@@ -5,8 +5,6 @@ git:
   depth: 10
 
 node_js:
-  - 6
```

Comment:
We should discuss, if we want to bump the major version first, because engines defined >= 6.0.0

Reply:
I will say this should be merged after we have bumped this repo for next major.

Since the engine is `>=6` in `package.json`, I feel that is telling people that the package supports  `>= 6` but for that to be guaranteed, it should be tested.

Since the end goal is to drop 6 and 8, the engine should be updated to `>= 10` and this is considered a breaking major change.

Reply:
Major bump is here: #91

Reply:
Major is bumped, resolving.